### DISCUSSION TITLE
Receiving WhatsApp notifications on the phone

### DISCUSSION THREAD

**Discussion by OrionEnes** [devlikeapro/waha#618] [Q&A]
Hello,
when starting a session, the notification does not appear on the phone. In the documentation at https://waha.devlike.pro/docs/engines/noweb/#configuration, you mentioned using 'markOnline: true,' but you didn’t provide complete documentation. We tried both 'markOnline: true' and 'markOnline: false,' but the notification still didn’t appear on the phone. How can we set this up correctly?

**Comment by devlikepro** [comment]
Hi! 
Yes, it should be`false`
https://github.com/devlikeapro/waha/issues/412#issuecomment-2357668483

Important thing to remember is to mark the connection back to 'offline" when you send a message via API as well:

> You need to send `offline` after all presence you send (WhatsApp doesn't send notifications to the device if a web client is active).

If that doesn't help - could you share your session POST request (if you don't start it via dashboard)?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by gamoridev** [reply]
@devlikepro What are the **all actions** that changes the presence, including sending a message?

I also have this problem mentioned by OP, even with _markOnline_ property setted to **false**.
At some point we stop receiving notifications and I don't know what is triggering this.
